### PR TITLE
chore: Adds info about next release

### PR DESCRIPTION
**Pull request by bk201-**
- New release info
- Fixes README typos

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
 # Change Log
 
+## next
+
+### Added
+
+- New UI for executing queries in the NoSQL account. UI supports viewing results in a table, tree or a JSON view, showing the query stats and the RU cost.
+- New UI for executing search queries in the vCore account. UI supports viewing results in a table, tree or a JSON view.
+- New UI for creating, viewing and editing documents in the separated tab.
```
Comment (sevoku) [inline CHANGELOG.md:9]:
```suggestion
- New UI for creating, viewing and editing documents in a separate tab.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+- New UI supports themes from VS Code (except Monarch).
```
Comment (sevoku) [inline CHANGELOG.md:10]:
```suggestion
- New UI supports VS Code theming (except Monarch).
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,30 @@
...
+
+- Jest for unit testing.
```
Comment (sevoku) [inline CHANGELOG.md:12]: CHANGELOG.md is rather customer facing (also for VS Marketplace), I don't think we need to mention these things here. Devs who are interested can review the Git history.

**Review by sevoku — COMMENTED** [review]
Thanks for getting this started!

File: CHANGELOG.md (hunk 3 above)
Comment (bk201-) [reply CHANGELOG.md]: Got it! Fixed

**Pull request merged**